glean_parser/templates/go_server.jinja2

```diff
@@ -0,0 +1,263 @@
+{# The final Go code is autogenerated, but this template is not. Please file bugs! #}
+package glean
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// AUTOGENERATED BY glean_parser v{{ parser_version }}. DO NOT EDIT.
+
+// required imports
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// log type string used to identify logs to process in the Moz Data Pipeline
+var gleanEventMozlogType string = "glean-server-event"
+
+// interface to send the log messages, accepting a call passing in the eventType and the message.
+// TODO: confirm what calls to cloud logging would be required to see if the log method signature makes sense
+// TODO: confirm if this should depend on https://github.com/mozilla-services/go-mozlog
+type Logger interface {
+	log(eventType string, message string)
+}
```

Comment:
I learned go-mozlog existed on Friday and wondered if that would be the expectation to use -- the JS implementation depends on a mozlog module, wheras the ruby one doesn't; but it does include fields similar to https://github.com/mozilla-services/go-mozlog/blob/master/mozlog.go#L70

It also isn't clear to me yet if would emit the logs to the right place for ingestion in the data pipeline. Does output to Stdout get collected in cloud logs by default?

Reply:
JS implementation uses mozlog because FxA (first customer) was using it. I'm not sure what's the standard approach for logging for new services now, but I'd say there's no need to bring in mozlog just for telemetry, it's fine if we just structure the log the same way as json.

> It also isn't clear to me yet if would emit the logs to the right place for ingestion in the data pipeline. Does output to Stdout get collected in cloud logs by default?

Yes, they should be if you'll be running in GCPv2 which I believe will be the case.